sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java

```diff
@@ -12,104 +17,200 @@
+            return new ExposureSummary[size];
+        }
+    };
+
     /**
      * The number of days since the last exposure (i.e. the last collected key resulting positive)
+     *
      * @see <a href="https://developers.google.com/android/exposure-notifications/exposure-notifications-api#dayssincelastexposure">documentation on developers.google.com</a>
      */
-    public int daysSinceLastExposure = -1;
+    public int getDaysSinceLastExposure() {
+        return daysSinceLastExposure;
+    }
 
     /**
      * The number of matched positive keys
+     *
      * @see <a href="https://developers.google.com/android/exposure-notifications/exposure-notifications-api#matchedkeycount">documentation on developers.google.com</a>
      */
-    public int matchedKeyCount = -1;
+    public int getMatchedKeyCount() {
+        return matchedKeyCount;
+    }
 
     /**
-     * The maximum risk score obained from all exposures
+     * The maximum risk score obtained from all exposures
+     *
      * @see <a href="https://github.com/corona-warn-app/cwa-app-android/blob/4906bd3b16959b1bca336c081a3718ac06f928c4/Corona-Warn-App/src/test/java/de/rki/coronawarnapp/storage/ExposureSummaryRepositoryTest.kt">used inside de/rki/coronawarnapp/storage/ExposureSummaryRepositoryTest.kt</a>
      * @see <a href="">documentation on developers.google.com</a>
      */
-    public int maximumRiskScore = -1;
+    public int getMaximumRiskScore() {
+        return maximumRiskScore;
+    }
 
     /**
      * Contains the cumulative duration in minutes the user spent in the three ranges of risk
      * defined by {@link ExposureConfiguration#durationAtAttenuationThresholds}. In particular:<br>
      * attenuationDurations[0]  |  sum of durations of exposures with an attenuation less than the
-     *                             low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])<br>
+     * low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])<br>
      * attenuationDurations[1]  |  sum of durations of exposures with an attenuation greater than
-     *                             equal to the low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])
-     *                             but less than the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
+     * equal to the low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])
+     * but less than the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
      * attenuationDurations[2]  |  sum of durations of exposures with an attenuation greater than
-     *                             or equal to the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
+     * or equal to the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
+     *
```

Comment:
Add back the spaces, these were here on purpose

Reply:
Spaces – or whitespace in general – don't translate well in JavaDoc. If anybody has to read the JavaDoc for that function without looking at the comment in the source code it looks like this: 
![Screen Shot 2020-09-29 at 11 03 11](https://user-images.githubusercontent.com/2513555/94537876-a2a94a00-0243-11eb-871f-9c751594c603.png)

I think we can both agree that that's also not the intended layout. I tried to make it readable both in "compiled" JavaDoc and the source code by converting it to a table, are you okay with the new format?

Reply:
BTW the new version would look like this:

![Screen Shot 2020-09-29 at 11 13 15](https://user-images.githubusercontent.com/2513555/94538855-d173f000-0244-11eb-9aa3-272023332dd2.png)

Reply:
Ok, great, thank you